ios/ReactNativePayments.m

```diff
@@ -430,14 +486,6 @@ - (void)handleUserAccept:(PKPayment *_Nonnull)payment
         paymentResponse[@"shippingContact"] = [self contactToString:payment.shippingContact];
     }
     
-    if (payment.billingContact) {
```

Comment:
How come these have been removed? Is it an API thing or because they're available elsewhere?

Want to make sure there isn't a breaking change.

Reply:
they're duplicated a couple of lines above. must have been some merge conflict that got into master previously: https://github.com/naoufal/react-native-payments/blob/224e5aba84b8d3ddbca84ee0597c71f74534b382/ios/ReactNativePayments.m#L425